logic/peer.js

```diff
@@ -146,7 +149,6 @@ Peer.prototype.parseInt = function (integer, fallback) {
  */
 Peer.prototype.applyHeaders = function (headers) {
 	headers = headers || {};
-	headers = this.normalize(headers);
 	this.update(headers);
```

Comment:
Are you sure that `this.update` will mutate `headers` object? If not - dropping that redundant `this.normalize` can result with an unwanted change of behavior.